uber/site_sections/registration.py

```diff
@@ -167,6 +167,18 @@ def history(self, session, id):
                                .order_by(Tracking.when).all()
         }
 
+    @ajax
+    def resend_email(self, session, id):
+        message = {''}
+        if id:
+            try:
+                email = session.query(Email).filter(Email.model == 'Attendee', Email.fk_id == id).all()[0]
+                send_email(c.ADMIN_EMAIL, email.dest, email.subject, email.html)
+                message = 'Email Sent!'
+            except ValueError:
+                message = 'No Emails Found With That ID'
```

Comment:
This is purely a stylistic thing, but most of our messages do not use title capitalization and instead read more like sentences (usually without periods at the end).